pkg/conciergeclient/conciergeclient.go

```diff
@@ -12,7 +12,7 @@ import (
 	"net/url"
 	"strings"
 
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
The changes under `pkg/conciergeclient` are just reverts right?

Reply:
Yep, that is correct.